virtcontainers/container.go

```diff
@@ -28,6 +28,23 @@ import (
 	"github.com/kata-containers/runtime/virtcontainers/utils"
 )
 
+var cDROMMajors = map[int64]string{
```

Comment:
The name looks weird, how about cdromMajors?

Also can you add some comments with reference links to this major definition? Then we can easily check if these are all :-)